mysql/metadata.csv

```diff
@@ -1,5 +1,6 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
 mysql.info.schema.size,gauge,,mebibyte,,"Size of schemas in MiB",0,mysql,mysql schema size
+mysql.info.table.size,gauge,,mebibyte,,"Size of tables in MiB",0,mysql,mysql table size
```

Comment:
To have precise metrics, it would be useful to have details of "data size per table" and "index size per table", but not just the sum of the 2.
So, maybe 2 metrics like:
- `mysql.info.table.index_size`
- `mysql.info.table.data_size`